web/src/pages/dashboard.tsx

```diff
@@ -162,15 +167,48 @@ const DashboardPage = (props: Props): ReactElement => {
     );
   };
 
+  const renderQuickActions = (): ReactElement => {
+    const renderQuickActions = [];
+
+    const registryUpdateBrcAmendment = props.quickActionTasks.find(
+      (e) => e.filename === "registry-update-brc-amendment"
+    );
+
+    if (registryUpdateBrcAmendment) {
+      renderQuickActions.push(
+        <QuickActionTaskTile
+          quickAction={registryUpdateBrcAmendment}
+          key={registryUpdateBrcAmendment.filename}
+        />
+      );
+    }
+
+    const nonHicStateContracting = props.quickActionLinks.find(
+      (e) => e.filename === "state-contracting-external-link"
+    );
+
+    if (nonHicStateContracting && !isHomeContractorIndustry(business?.profileData.industryId)) {
```

Comment:
[pebble] thoughts on extracting the second half of the conditional into it's own variable so it's easier to read? 
`if(nonHicStateContracting && isNotHomeContractor)`

Reply:
or even extracting the whole conditional and naming it something along the lines of `renderNonHicStateContracting`so it's `if(renderNonHicStateContracting){...}`